### ISSUE TITLE
[Feature equest] Compatibility with group folder

### ISSUE THREAD

**Issue by tanguy-opendsi** [nextcloud/files_downloadactivity#70]
Hi
It is possible make compatible with group folder app ? [https://github.com/nextcloud/groupfolders](https://github.com/nextcloud/groupfolders) 

Best regards

**Comment by Mululu** [comment]
Any updates on this enhancement?

**Comment by nickvergessen** [comment]
Not really, the app is more in a maintenance situation at the moment as my own usecase disappeared and my available coding time is limited
